packages/renderer/src/lib/recommendation/ExtensionBanner.svelte

```diff
@@ -22,6 +25,25 @@ $effect(() => {
     style = undefined;
   }
 });
+
+async function onClose(): Promise<void> {
+  let result: MessageBoxReturnValue = { response: -1 };
+  try {
+    result = await window.showMessageBox({
+      title: 'Hide extension recommendation banners',
+      message: `Do you want to hide extension recommendation banners?`,
+      type: 'warning',
+      buttons: [`No, keep them`, 'Yes, hide'],
```

Comment:
![image](https://github.com/user-attachments/assets/f902b4b0-8670-4270-af97-1c8569450281)

Should we invert the buttons ? Look odds to me